### ISSUE TITLE
`make all` fails on Ubuntu 24.04

### ISSUE THREAD

**Issue by jd-develop** [vlang/vinix#155]
Once Jinx cache has been downloaded, a Permission denied error occurs. Here’s the relevant output:

```
% Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  109M  100  109M    0     0   809k      0  0:02:18  0:02:18 --:--:-- 1060k
/path/to/vinix/.jinx-cache/rbrt: failed to open or write to /proc/self/setgroups at line 186: Permission denied
```

I tried with `make all` and `PKGS_TO_INSTALL='*' make all`: both failed.

I’m on commit `6d68d67471c65e6e3b7dc8b5547a432295680c59` (tag `weekly-2024-10-19`)

**Comment by jd-develop** [comment]
Update: I manually edited `jinx` to know if it failed to open or to write in `/proc/self/setgroups`: it fails to open it.

**Comment by mintsuki** [comment]
Hi, sorry for the late reply. I documented how to fix this in commit `86d5879`, thanks for reporting!

**Comment by jd-develop** [comment]
Thank you :)
